.gitignore

```diff
@@ -64,6 +64,7 @@ Dynamo.sln.DotSettings
 *.sdf
 *.backup
 *.pdf
+*.dgml
```

Comment:
Just curious, what is this exception here for?

Reply:
I was experimenting with the new IntelliTrace feature in VS 2019. This file is generated using that tool, I thought it would be good to exclude it as it's generated by VS.